MainActivity.java

```diff
@@ -0,0 +1,18 @@
+package com.soumyajyoti.firstapplication;
+
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Toast;
+
+public class MainActivity extends AppCompatActivity {
+    public void fun1(View view){
+        Toast.makeText(this, "This is a sample App", Toast.LENGTH_SHORT).show();
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
```

Comment:
You need to put a comment here, what this function is doing.
All functions should have a comment above, as to what the function is doing.

Reply:
The comment should be outside the function, just above the function. Not at the place where you have written.